src/views/Home/index.tsx

```diff
@@ -71,6 +78,29 @@ const HomeView = (): JSX.Element => {
     }
   }, [categories])
 
+  useEffect(() => {
+    handleGetPacks();
```

Comment:
no need for asynchronous / await call?

Reply:
I could see if I can add a loading skeleton to this, otherwise it doesn't make much sense to block here since this can basically run in the background whilst the user enters their search query